AEPCore/Tests/Data+HexStringTests.swift

```diff
@@ -0,0 +1,36 @@
+/*
+ Copyright 2020 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+@testable import AEPCore
+import XCTest
+
+class Data_HexStringTests: XCTestCase {
+
+    /// Tests that the string is properly encoded
+    func testHexStringFromDataHappy() {
```

Comment:
let's add a test which covers all hex characters `0123456789ABCDEF`

Reply:
Also, we should consider adding migration tests for push tokens to make sure they don't change (ACP->AEP).

Reply:
We are not currently migrating the push ID as we don't migrate it from v4 ->v5. Since the customer usually sets push id on every launch I don't think it needs to be migrated.

I've added a longer test which should cover all the hex characters.